recipes/pthreadpool/all/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_library(fxdiv INTERFACE)
+target_link_libraries(fxdiv INTERFACE CONAN_PKG::fxdiv)
+
+if(MSVC AND BUILD_SHARED_LIBS)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif()
```

CMake handles the logic

```suggestion
set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```